lib/internal/timers.js

```diff
@@ -555,7 +556,7 @@ function getTimerCallbacks(runNextTicks) {
         if (args === undefined)
           timer._onTimeout();
         else
-          timer._onTimeout(...args);
+          timer._onTimeout(...new SafeArrayIterator(args));
```

Comment:
This is probably more performant:
```suggestion
          ReflectApply(timer._onTimeout, timer, args);
```